printer_mods/Simone67/TornadoAirFlow/README.MD

```diff
@@ -0,0 +1,36 @@
+<H1>TORNADO AIR FLOW for AfterBurner</H1><BR>
+  
+  ![one Side View](./IMAGE/sez_B.png)
+
+  Using two 5015 fans to get very powerful workpiece cooling air flow and necessary to get quality prints when using PLA material which needs strong air flow to cool immediately.
+  ![one Side View](./IMAGE/sez_A.png)
+  ![one Side View](./IMAGE/sez_C.png)
+
+  
+## IMPROVING PLA PRINT QUALITY
+![one Side View](./IMAGE/compare.jpg)
+First one printed with low air flow, second one using Tornado Air Flow at 100% power
+
+  
+## Screenshot
+   ![one Side View](./IMAGE/screenshot.jpg) 
+  
+## Printing
+  
+Tested with recommended settings for Voron parts:Layer 0.2 - ABS, 4 perimeters, 5 tops/bottoms, 40% infill.
+All STL files are correctly oriented and do not need support.
+STL were tested by printing and assembling !!!
+  
+## Printing Orientation
+  ![one Side View](./IMAGE/orientation.jpg) 
+
+  ## Hardware
+  
+Qty | Item
```

Comment:
While Github appears to render this, please confirm to proper Markdown syntax and put the vertical lines at the start and end of each table row.